src/api/controllers/db-controller.ts

```diff
@@ -468,34 +478,58 @@ export interface GetTxWithEventsArgs extends GetTxArgs {
   eventOffset: number;
 }
 
+function isDroppedTx(tx: DbMempoolTx): boolean {
```

Comment:
Can tell the compiler the result of this is a specific type of DbMempoolTx, with `val is XXX` return syntax